coders/composio_coders/cli.py

```diff
@@ -0,0 +1,147 @@
+        repo_name = path.parent.name
+        return repo_name
+    except git.InvalidGitRepositoryError:
+        return None
+
+
+@click.command(name="setup", help="🔑 Setup model configuration in the current directory")
+def setup():
+    """Setup model configuration in the current directory."""
+    config = {}
+    model_env = click.prompt("🔑 Choose the model environment to be used for initiating agent", 
+                             type=click.Choice(['openai', 'azure'], case_sensitive=False))
+
+    if model_env == "openai":
+        config["model_env"] = "openai"
+        api_key = click.prompt("🔑 Please enter openai API key", type=str)
+        config["api_key"] = api_key
+    if model_env == "azure":
+        config["model_env"] = "azure"
+        api_key = click.prompt("🔑 Please enter azure key", type=str)
+        endpoint_url = click.prompt("🌐 Please enter Azure endpoint URL", type=str)
+        config["api_key"] = api_key
+        config["endpoint_url"] = endpoint_url
+
+    config_path = Path(MODEL_ENV_CONFIG_PATH)
+    with config_path.open('w') as f:
+        json.dump(config, f)
+    click.echo(f'🍀 Model configuration saved to {MODEL_ENV_CONFIG_PATH}')
+
+
+@click.command(name="add_issue", help="➕ Add an issue configuration to the current directory")
+def add_issue():
+    """Add an issue configuration to the current directory."""
+    curr_repo_name = get_git_root()
+    repo_name = click.prompt("Enter the repo name to start solving the issue", type=str, default="", show_default=False)
+    if not repo_name or not repo_name.strip():
+        if curr_repo_name:
+            click.echo(f"no git repo-given. Initializing git repo from current directory: {curr_repo_name}")
+            repo_name = curr_repo_name
+        else:
+            click.echo("❗!! Error: no git repo found or given. Exiting setup...")
+            return
+    issue_id = click.prompt("Please enter issue id", type=str)
+    base_commit_id = click.prompt("Please enter base commit id", type=str, default="", show_default=False)
+    issue_description = click.prompt("Please enter issue description", type=str)
+    issue_config = {
+        'repo_name': repo_name,
+        'issue_id': issue_id,
+        'base_commit_id': base_commit_id,
+        'issue_description': issue_description
+    }
+    config_path = Path(ISSUE_CONFIG_PATH)
+    with config_path.open('w') as f:
+        json.dump(issue_config, f)
+    click.echo(f'🍀 Issue configuration saved to {ISSUE_CONFIG_PATH}\n')
+
+
+@click.command(name="solve", help="👷 Start solving the configured issue")
+def solve():
+    """Start solving the configured issue."""
+    git_access_token = os.environ.get(KEY_GIT_ACCESS_TOKEN)
+    if not git_access_token or not git_access_token.strip():
+        click.echo(f"❗ Error: {KEY_GIT_ACCESS_TOKEN} is not set in the environment.\n")
+        click.echo(f"🔑 Please export your GIT access token: {KEY_GIT_ACCESS_TOKEN} and try again !")
+        return
```

Comment:
This should be done in the `setup` command not when running the agent

Reply:
fixed